scripts/build_images/build_multiplatform.py

```diff
@@ -0,0 +1,77 @@
+import os
+import tempfile
+
+from build import build_images
+from combine import combine_images_from_metadata_dir
+from utils import (
+    buildx_builder,
+    get_current_platform,
+    get_env_or_exit,
+    local_registry,
+    print_and_run_command,
+)
+
+# The container name for the local Docker registry.
+REGISTRY_NAME = "prairielearn-registry"
+
+# The name for the `buildx` builder to create and use.
+BUILDER_NAME = "prairielearn-builder"
+
+
+def get_platforms() -> list[str]:
+    platforms = os.environ.get("PLATFORMS")
+    if platforms is None:
+        return [get_current_platform()]
+
+    # Split the platforms by comma and strip whitespace.
+    return [platform.strip() for platform in platforms.split(",")]
+
+
+if __name__ == "__main__":
+    images = get_env_or_exit("IMAGES")
+    tag = get_env_or_exit("TAG")
+    platforms = get_platforms()
+
+    # Note that base images must come first in the list so that they're built first.
+    image_list = images.split(",")
```

Comment:
whitespace is stripped on platforms, but not images -- we should be consistent.

Reply:
Made consistent in 0bf8506 (stripping shouldn't be necessary).